From: Michal Kubecek <mkubecek@suse.cz>
Date: Sun, 14 Jul 2019 20:58:33 +0200
Subject: Kconfig: make CONFIG_CC_CAN_LINK always true
Patch-mainline: Never, SUSE specific
References: bsc#1127188

SUSE kernels are always built in an environment where the toolchain can
link userspace programs but run_oldconfig.sh and kbuild checks use cross
compilers for non-x86 programs so that they would end up with wrong value
of CONFIG_CC_CAN_LINK.

Make CONFIG_CC_CAN_LINK always true to work around this until we find
a better solution.

Signed-off-by: Michal Kubecek <mkubecek@suse.cz>
---
 init/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/init/Kconfig
+++ b/init/Kconfig
@@ -25,7 +25,7 @@ config CLANG_VERSION
 	default $(shell,$(srctree)/scripts/clang-version.sh $(CC))
 
 config CC_CAN_LINK
-	def_bool $(success,$(srctree)/scripts/cc-can-link.sh $(CC))
+	def_bool y
 
 config CC_HAS_ASM_GOTO
 	def_bool $(success,$(srctree)/scripts/gcc-goto.sh $(CC))
